draw/src/main/res/layout/item_view_option.xml

```diff
@@ -5,13 +5,12 @@
     android:layout_height="wrap_content"
     android:drawablePadding="@dimen/margin_small"
     android:gravity="center_vertical"
-    android:minHeight="@dimen/touch_target_size"
+    android:minHeight="48dp"
```

Comment:
Ah good point. I'm not sure this should use that dimension as I think they only "happen" to be the same rather than being intended. I actually tried to remove the min height here, but something to do with the way this all works means that it needs it. I'm guessing we'll actually get rid of this layout at some point as we moved away from the `IconMenuListAdapter` for form quitting when it was redesigned and I reckon we probably will here eventually as well? Using a `ListView` for this feels overly complex. I could just leave that 48dp in there and deprecate the `Adapter` so it's clear this is code we're getting rid of?